packages/plugin-snowplow/lib/index.ts

```diff
@@ -1,18 +1,37 @@
 /* eslint-disable no-unused-vars, class-methods-use-this, import/no-unresolved */
 import {
-  Event, Properties, RequestLoggerPlugin, PluginLoadOptions,
+  Event, PageOptions, TrackOptions, Properties, RequestLoggerPlugin, PluginLoadOptions, ResponseLogger,
 } from '@itly/sdk';
 
 export type SnowplowOptions = {
   url: string;
   config?: {};
 };
 
+export interface SnowplowContext {
+  schema: string;
+  data: { [key: string]: any };
+}
+
+export type SnowplowCallback = (...args: any[]) => void;
+
+export interface SnowplowCallOptions {
+  callback?: SnowplowCallback;
+  options?: {
```

Comment:
It seems like [Snowplow accepts contexts as a standalone argument](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-tracker/tracking-specific-events/#custom-contexts). I think to match their interface we move contexts up up off of options. Do you agree?

For Segment I added options to match `options.integrations` in [the Segment docs](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#track).

Reply:
Agree. Fixed